butterfly-slack/src/main/resources/postSuccess.json

```diff
@@ -0,0 +1,27 @@
+{
+  "channel": "%s",
+  "text": "Your Butterfly transformation concluded successfully",
+  "attachments": [
+    {
+      "text": "",
+      "color": "#cb42f4",
+      "actions": [
+        {
+          "text": "Transformation Details",
+          "type": "button",
+          "url": "https://butterfly.pp-devcos-butterfly.us-central1.gcp.dev.paypalinc.com/transformations/3575",
```

Comment:
Interpolate this with the actual link (I know you already plan to do it, but I am just commenting so we keep track of this pending change)